vendor/github.com/jacobsa/gcloud/gcs/bucket.go

```diff
@@ -210,13 +267,60 @@ func (b *bucket) ListObjects(
 	if listing, err = toListing(rawListing); err != nil {
 		return
 	}
+	fmt.Println((*listing))
```

Comment:
Should we have this statement? Or this was added for debugging?

Reply:
Sorry this was for debugging. Removed.